cookies.go

```diff
@@ -11,3 +11,7 @@ type Cookies struct {
 func (c *Cookies) Cookie(name string) (*http.Cookie, error) {
```

Comment:
maybe `Get` will better than `Cookie`?

Reply:
In fact, I prefer `cookie["name"]`, would it be too tricky?

Reply:
`cookie["name"]` like it can set value.
I think it was a little tricky.

Reply:
changed at: 130910c